### PR TITLE
13.0 include today dam

### PR DESCRIPTION
**Pull request by Polymorphe57**
Fix for the bug reported in https://github.com/odoo/odoo/pull/96239

### CODE REVIEW

File: addons/web/static/lib/py.js/lib/py_extras.js [hunk 1]
```diff
@@ -556,7 +556,7 @@ datetime.datetime = py.type('datetime', null, {
             }));
     },
     to_utc: function () {
-        var d = new Date(this.year, this.month, this.day, this.hour, this.minute, this.second);
+        var d = new Date(this.year, this.month -1, this.day, this.hour, this.minute, this.second);
```
Comment (Jinjiu96) [inline addons/web/static/lib/py.js/lib/py_extras.js:559]: Hello, I'm not sure why we need to edit the month here, is a special calculation related to `to_utc`?

Comment (Polymorphe57) [reply addons/web/static/lib/py.js/lib/py_extras.js:559]: You can have a look to https://github.com/odoo/odoo/pull/103353/commits/1b83d21675c393078830b482980d94e0aed555e0

Comment (Polymorphe57) [reply addons/web/static/lib/py.js/lib/py_extras.js:559]: I am going to fix this in 13.0  and add a test. You can remove it for now. If you don't do that and work in the timezone, you will see that the dates generated in python or js are different for this month :-)

Comment (Jinjiu96) [reply addons/web/static/lib/py.js/lib/py_extras.js:559]: Thanks for the explaination! :smile:

**Comment by Polymorphe57** [comment]
After some discussion with AAB, the solution seems inappropriate. It also targets a version that is no longer supported.

**Pull request closed without merging**